lib/routes/account.js

```diff
@@ -1747,6 +1765,8 @@ module.exports = function (
         const code = Buffer(request.payload.code, 'hex')
         const service = request.payload.service || request.query.service
         const reminder = request.payload.reminder || request.query.reminder
+        var isEmailVerification = false, isAccountVerification = false, account
```

Comment:
also using `var` here. If the value is set (`false` in this case) that I think we want to make sure they are on the single line and not listed with a `,`